tests/unit/coloredBookmarks.unit.test.js

```
@@ -0,0 +1,401 @@
+		expect(resolveColorAlias('9')).toBeNull();
+		expect(resolveColorAlias(null)).toBeNull();
+	});
+	it('trims whitespace', () => {
+		expect(resolveColorAlias('  amber  ')).toBe(1);
+	});
+});
+
+describe('getColorForVerse + helpers with store', () => {
+	beforeEach(() => {
+		__userColoredBookmarks.set({});
+		(globalThis.localStorage || window.localStorage)?.clear?.();
```

ESLint

[error] 'globalThis' is not defined.

(no-undef)